pkg/stringutil/stringutil.go

```diff
@@ -14,13 +14,20 @@
 
 package stringutil
 
-import "math/rand"
+import (
+	"math/rand"
+	"time"
+)
 
 const (
 	chars = "abcdefghijklmnopqrstuvwxyz0123456789"
 )
 
-// UniqueStrings retruns a slice of randomly generated unique strings.
+func init() {
```

Comment:
This will override rand seed setting for anything that imports this package. It should the be importer's responsibility to seed rand, so don't to this in init please.